rust_src/remacs-macros/README.md

```diff
@@ -0,0 +1,44 @@
+remacs-macros
+=============
+
+# Attributes
+
+## `lisp_fn`
+This macros creates the necessary functions and symbols automatically.
+It handles `MANY` functions and normal functions too (support for `UNEVALLED`
+needs to be done).
+
+It is used like this:
+
+```rust
+/// Return the same argument
+#[lisp_fn(name = "same", min = "1")]
+fn same(obj: LispObject) -> LispObject {
+    obj
+}
+```
+
+Where the `name` argument specifies the **symbol name** that is going to be use in Emacs Lisp, `min` specifies the **minimum** number of arguments that can be passed to this function.
+
+In this example the attribute generates the `Fsame` function that is going to b called in **C**, and the `Ssame` structure that holds the function information. You still need to register the function with `defsubr` to make it visible in Emacs Lisp. To make a function visible to C you need to export it in the crate root (lib.rs) as follows:
+
+```rust
+use somemodule::Fsome;
+```
+
+### `MANY` functions
+
+This attributes handles the definitions of `MANY` functions, these functions can take an arbitrary number of arguments, but you *still can* specify a `min` number of arguments.
```

Comment:
*This attribute

But I'm not sure what attribute you're referring too here. Do we need to use the term `MANY`? I think it's just a term the Emacs C uses, but readers of the README might not know it.